packages/@remirror/extension-annotation/src/types.ts

```diff
@@ -11,6 +11,16 @@ export interface AnnotationOptions<A extends Annotation = Annotation> {
    * the amount of annotations in a segment.
    */
   getStyle?: GetStyle<A>;
+
+  /**
+   * Allows to format the text returned for each annotation.
+   *
+   * When `blockSeparator` is given, it will be inserted whenever a new
+   * block node is started.
+   *
+   * @see PromirrorNode.textBetween
+   */
+  blockSeparator?: string | undefined;
```

Comment:
To allow undefined as a value, the following needs to be added.

Add this to the top of the file.

```ts
import { AcceptUndefined } from 'remirror/core';
```

```suggestion
  blockSeparator?: AcceptUndefined<string>;
```